app/index.js

```diff
@@ -205,6 +228,13 @@ app.on('ready', () => {
       }
     });
 
+    // Keep track of focus time of every window, to figure out
+    // which one of the existing window is the last focused.
+    // Works nicely even if a window is closed and removed.
+    win.on('focus', () => {
```

Comment:
Or `browser-window-focus`, but I've some concerns about this solution: I not sure it would handle closed window as well as uptime solution. What should happen if I've two opened windows, i close one without focusing the other? Or if I launch an hyperterm instance without focusing the window? (not all window manager behave the same throughout different systems).
On the other hand process.uptime check is absolutely rock solid (I can't figure out a situation where it could fail): if a window is closed the previous one will be returned by `app.getLastFocusedWindow` for sure.
And adding a simple integer property to win object is not a big deal :)